prism-agent/service/server/src/main/scala/io/iohk/atala/connect/controller/ConnectionEndpoints.scala

```diff
@@ -124,7 +137,13 @@ object ConnectionEndpoints {
       .out(
         jsonBody[ConnectionsPage].description("The list of connection flow records available from the agent's database")
       )
-      .errorOut(basicFailuresAndForbidden)
+      .errorOut(
```

Comment:
I wonder what is wrong with the previous definition of the errorOut?

Reply:
My two cents is that this is more verbose and easy to see what the actual failures are without needing to redirect. I remembert working on the Issuance controllers and kept needing to click around to find out why the OAS spec wasn't generated as I expected it

Reply:
Yes, this was precisely the goal. Being verbose to gain visibility on the exact list of errors returned by each endpoint.